README.md

```diff
@@ -17,7 +17,7 @@ The CLI is designed for authenticating and returning an access token for public
 | ------------------------------------------ | ----------------------- | ------------- | ---------------- | ------------- | ------------------------------- |
 | Windows                                    | ✅                      | ✅            | ✅               | ✅            | ⚠️ `--domain` account filtering |
 | OSX (MacOS)                                | ⚠️ via Web Browser      | ✅            | ✅               | ✅            | ⚠️ `--domain` account filtering |
-| Ubuntu (linux) <br/>‼️Releases coming soon | ⚠️ via Edge             | ✅            | ✅               | ✅            | ⚠️ `--domain` account filtering |
+| Ubuntu (linux) <br/>‼️Releases coming soon | ⚠️ via Edge             | ✅            | ✅               | ❎         | ⚠️ `--domain` account filtering |
```

Comment:
Is there a red X that we can use? I almost didn't notice that it's an X.

Reply:
Totally agreed! We couldn't find a red X :(

Reply:
Would this one do okay? ❌